src/main/groovy/com/hcl/security/appscan/gradle/tasks/SecurityTask.groovy

```diff
@@ -19,13 +19,17 @@ abstract class SecurityTask extends DefaultTask {
 
     private static Collection<ITarget> m_targets = new HashSet<ITarget>();
 
+    private static boolean isAddedToPostBuildAction = false;
```

Comment:
Can you rename this variable to something shorter, like "m_actionAdded" and use the "m_" prefix to match the convention for member variables in this project?